ui/src/main/resources/PhenoTips/SimilarCases.xml

```diff
@@ -318,6 +319,8 @@
       }
       data.featuresCount = data.query.features ? data.query.features.size() : 0;
 
+      data.resultsCount = Math.min(data.resultsCount, this._MAX_RESULTS_DISPLAYED);
+      data.results = data.results.slice(0, data.resultsCount);
```

Comment:
Good point. No, there's no guarantee. We should sort the results before paginating.